playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -229,3 +229,56 @@
   tasks:
   - include: docker/upgrade.yml
     when: l_docker_upgrade is defined and l_docker_upgrade | bool and not openshift.common.is_atomic | bool
+
+- name: Drain and upgrade master nodes
+  hosts: oo_masters_to_config:&oo_nodes_to_upgrade
```

Comment:
I would keep it there to stress we are upgrading masters that are nodes as well. Which label variable do you have in mind? The serial should be set to 1 always. I can do that in this PR or make it happen in another PR once I move the upgrade node play into the master upgrade loop.